src/server/qgsrequesthandler.h

```diff
@@ -96,33 +96,26 @@ class QgsRequestHandler
     //! @note not available in Python bindings
     virtual void setGetCoverageResponse( QByteArray* ba ) = 0;
 
-    virtual void setDefaultHeaders() {}
-
     //! Set an HTTP header
     virtual void setHeader( const QString &name, const QString &value ) = 0;
 
     //! Remove an HTTP header
-    virtual int removeHeader( const QString &name ) = 0;
-
-    //! Delete all HTTP headers
-    virtual void clearHeaders() = 0;
+    virtual void removeHeader( const QString &name ) = 0;
 
     //! Append the bytestream to response body
     virtual void appendBody( const QByteArray &body ) = 0;
 
-    //! Clears the response body
-    virtual void clearBody() = 0;
```

Comment:
that made it possible to create/edit/delete individual headers, the current implementation only misses the ability to get the list of header names and retrieve a single one, which should be added now.